reference/debugging.md

```diff
@@ -0,0 +1,229 @@
+
+To see a demonstration of how you can debug your Ambassador instance, follow the instructions to create an example mapping configuration.
+
+Note: The following assumes that you deployed Ambassador and the following services from the [quick start installation guide](../../user-guide/install) to a Kubernetes cluster.
+
+**In the command line:**
+
+1. Create a cluster in GKE with RBAC support enabled, and your user account configured correctly. Then run the following:
+
+    ```console
+    $ gcloud container clusters create ambassador-demo --preemptible
+    $ kubectl create clusterrolebinding cluster-admin-binding-new \
+    --clusterrole cluster-admin --user <your_user_name>
+    ```
+
+2. Deploy the latest version of Ambassador following the instructions from [here](../../user-guide/install).
+3. Next, create an Ambassador Service and deploy a basic `httpbin` Ambassador Mapping by saving the following YAML to a file named `ambassador-services.yaml`
+
+   ```yaml
+    ---
+    apiVersion: getambassador.io/v2
+    kind:  Mapping
+    metadata:
+      name:  httpbin
+    spec:
+      prefix: /httpbin/
+      service: httpbin.org
+      host_rewrite: httpbin.org
+    ```
+
+4. Apply this into your cluster with the following command:
+
+    ```console
+    $ kubectl apply -f ambassador-services.yaml
+    ```
+
+You should now be able to utilize this example mapping for debugging demonstrations and purposes.
+
+## Check Ambassador Status
+
+First, check to see if the [Diagnostics](../diagnostics) service is reachable. The following command assumes you have installed Ambassador in the `ambassador` name space: `kubectl port-forward -n ambassador_<pod name> 8877`
+
+If it is successful, try to diagnose your original issue with the Diagnostics Console.
+
+**If it is not successful, complete the following to see if Ambassador is running:**
+
+1. Check the Ambassador deployment with the following: `kubectl get deployments`
+2. After a brief period, the terminal will print something similar to the following:
+
+    ```console
+    $ kubectl get deployments
+    NAME         DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+    ambassador   3         3         3            3           1m
+    ```
+
+3. Check that the “desired” number of Pods equals the “current” or “available” number of pods. If they are **not** equal, check the status of the associated pods with the following command: `kubectl get pods`
+
+    The terminal should print something similar to the following:
+
+    ```console
+    $ kubectl get pods
+    NAME                         READY     STATUS    RESTARTS   AGE
+    ambassador-85c4cf67b-4pfj2   1/1       Running   0          1m
+    ambassador-85c4cf67b-fqp9g   1/1       Running   0          1m
+    ambassador-85c4cf67b-vg6p5   1/1       Running   0          1m
+    ```
+
+4. If any of the Pods have a status of “not started,” use the following command to “describe” the Deployment pods: `kubectl describe deployment ambassador`
```

Comment:
Maybe "...use the following command to get details about the history of the Deployment:" ?

```
kubectl describe -n ambassador deployment ambassador
```